packages/platform-android/src/commands/runAndroid/tryRunAdbReverse.ts

```diff
@@ -4,15 +4,17 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
 import {execFileSync} from 'child_process';
 import {logger} from '@react-native-community/cli-tools';
 import getAdbPath from './getAdbPath';
 
 // Runs ADB reverse tcp:8081 tcp:8081 to allow loading the jsbundle from the packager
-function tryRunAdbReverse(packagerPort: number | string, device?: string) {
+function tryRunAdbReverse(
+  packagerPort: number | string,
+  device?: string | void,
```

Comment:
```suggestion
  device?: string,
```

Reply:
This also isn't working, as we sometimes pass `undefined` to the function.